src/mixed.js

```diff
@@ -48,7 +48,12 @@ const addToList = (value, list) => {
 const hasInList = (value, resolve, list) => {
   if (list.has(value)) return true;
 
-  let item; let values = list.refs.values()
+  let item;
```

Comment:
maybe simplify this a bit and add `if (!this.refs) return false` above this line so we return early for these cases?